src/formControl.js

```diff
@@ -0,0 +1,35 @@
+const formElem = document.querySelector('#addPorts');
+formElem.addEventListener('submit', (e) => {
```

Comment:
Id add a clear line here to separate the larger function from the declaration above it
```
const formElem = document.querySelector('#addPorts');

formElem.addEventListener('submit', (e) => {
```